keyboards/kuro/kuro65/info.json

```
@@ -0,0 +1,191 @@
+            { "flags": 1, "matrix": [2,  0], "x":  6, "y": 32}, 
+            { "flags": 4, "matrix": [2,  1], "x": 26, "y": 32}, 
+            { "flags": 4, "matrix": [2,  2], "x": 41, "y": 32}, 
+            { "flags": 4, "matrix": [2,  3], "x": 56, "y": 32}, 
+            { "flags": 4, "matrix": [2,  4], "x": 71, "y": 32}, 
+            { "flags": 4, "matrix": [2,  5], "x": 86, "y": 32}, 
+            { "flags": 4, "matrix": [2,  6], "x":101, "y": 32}, 
+            { "flags": 4, "matrix": [2,  7], "x":116, "y": 32}, 
+            { "flags": 4, "matrix": [2,  8], "x":131, "y": 32}, 
+            { "flags": 4, "matrix": [2,  9], "x":146, "y": 32}, 
+            { "flags": 4, "matrix": [2, 10], "x":161, "y": 32}, 
+            { "flags": 4, "matrix": [2, 11], "x":176, "y": 32}, 
+            { "flags": 4, "matrix": [2, 12], "x":191, "y": 32}, 
+            { "flags": 4, "matrix": [2, 14], "x":208, "y": 24}, 
+            { "flags": 4, "matrix": [2, 15], "x":224, "y": 32}, 
+            { "flags": 4, "matrix": [1, 15], "x":224, "y": 16}, 
+            { "flags": 4, "matrix": [1, 12], "x":188, "y": 16}, 
+            { "flags": 4, "matrix": [1, 11], "x":173, "y": 16}, 
+            { "flags": 4, "matrix": [1, 10], "x":158, "y": 16}, 
+            { "flags": 4, "matrix": [1,  9], "x":143, "y": 16}, 
+            { "flags": 4, "matrix": [1,  8], "x":128, "y": 16}, 
+            { "flags": 4, "matrix": [1,  7], "x":113, "y": 16}, 
+            { "flags": 4, "matrix": [1,  6], "x": 98, "y": 16}, 
+            { "flags": 4, "matrix": [1,  5], "x": 83, "y": 16}, 
+            { "flags": 4, "matrix": [1,  4], "x": 68, "y": 16}, 
+            { "flags": 4, "matrix": [1,  3], "x": 53, "y": 16}, 
+            { "flags": 4, "matrix": [1,  2], "x": 38, "y": 16}, 
+            { "flags": 4, "matrix": [1,  1], "x": 23, "y": 16}, 
+            { "flags": 1, "matrix": [1,  0], "x":  4, "y": 16}, 
+            { "flags": 1, "matrix": [0,  0], "x":  0, "y":  0}, 
+            { "flags": 4, "matrix": [0,  1], "x": 15, "y":  0}, 
+            { "flags": 4, "matrix": [0,  2], "x": 30, "y":  0}, 
+            { "flags": 4, "matrix": [0,  3], "x": 45, "y":  0}, 
+            { "flags": 4, "matrix": [0,  4], "x": 60, "y":  0}, 
+            { "flags": 4, "matrix": [0,  5], "x": 75, "y":  0}, 
+            { "flags": 4, "matrix": [0,  6], "x": 90, "y":  0}, 
+            { "flags": 4, "matrix": [0,  7], "x":105, "y":  0}, 
+            { "flags": 4, "matrix": [0,  8], "x":120, "y":  0}, 
+            { "flags": 4, "matrix": [0,  9], "x":135, "y":  0}, 
+            { "flags": 4, "matrix": [0, 10], "x":150, "y":  0}, 
+            { "flags": 4, "matrix": [0, 11], "x":165, "y":  0}, 
+            { "flags": 4, "matrix": [0, 12], "x":180, "y":  0}, 
+            { "flags": 4, "matrix": [0, 14], "x":203, "y":  0}, 
+            { "flags": 4, "matrix": [0, 15], "x":224, "y":  0}
+        ]
+    },
+    "bootmagic": { 
+        "enabled": true,
+        "matrix": [0, 0]
+    }
```

This is the default behaviour
```suggestion
    }
```

---

Thanks! I was not sure, if I need this section.